examples/taskruns/cloud-event.yaml

```diff
@@ -75,6 +74,30 @@ spec:
       - name: notification
         type: cloudEvent
   steps:
+    - name: wait-for-sink
+      image: python:3-alpine
+      imagePullPolicy: IfNotPresent
```

Comment:
This is the default, so I think we can just omit it for brevity.

Reply:
nice catch, thanks